TechnologyAssessmentApp/TechnologyAssessmentApp/POP Pattern/MainViewController.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  MainViewController.swift
+//  TechnologyAssessmentApp
+//
+//  Created by Kshitij Godara on 24/08/18.
+//  Copyright © 2018 Nagarro. All rights reserved.
+//
+
+import UIKit
+
+class FoodImageView: UIImageView, Shakeable {
+    // other customization here
+}
+
+class MainViewController: UIViewController, ShakeDeviceProtocol {
+
+    @IBOutlet weak var foodImageView: FoodImageView!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+    }
+
+    @IBAction func shakeImagePressed(_ sender: UIButton) {
+        foodImageView.shake()
+    }
+    override internal var canBecomeFirstResponder: Bool {
+        return true
+    }
+    override func motionEnded(_ motion: UIEventSubtype, with event: UIEvent?) {
```

Comment:
why you don't move this to `Shakable` protocol instead ? It's the entire purpose of this activity ! Like that every VC who are implementing a `Shakable` protocol will automatically show an alert when the phone is being shake by user !